library/Potoki/Core/IO.hs

```diff
@@ -1,11 +1,18 @@
-module Potoki.Core.IO where
+module Potoki.Core.IO (
+  module Fetch,
+  produceAndConsume,
+  produceAndTransformAndConsume,
+  produce,
+  consume,
+  transformList,
+) where
 
 import Potoki.Core.Prelude
 import Potoki.Core.Types
 import qualified Potoki.Core.Produce as A
 import qualified Potoki.Core.Consume as B
 import qualified Acquire.IO as C
-
+import qualified Potoki.Core.IO.Fetch as Fetch
```

Comment:
Quite a misleading choice of name for the alias. In case you don't like the "A, B, C" convention, at least name it more clearly like "FetchIO".

Alternatively you could have just imported it into the global scope and directly reexported the function.